tests/kuttl/tests/dataplane-deploy-multiple-secrets/00-assert.yaml

```diff
@@ -0,0 +1,382 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: generic-service1
+spec:
+  caCerts: combined-ca-bundle
+  tlsCert:
+    contents:
+    - dnsnames
+  play: |
+    - hosts: localhost
+      gather_facts: no
+      name: kuttl play
+      tasks:
+        - name: Sleep
+          command: sleep 1
+          delegate_to: localhost
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: default-install-certs-override
+spec:
+  addCertMounts: True
+  play: |
+    - hosts: localhost
+      gather_facts: no
+      name: kuttl play
+      tasks:
+        - name: Sleep
+          command: sleep 1
+          delegate_to: localhost
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: openstack-edpm-tls
+spec:
+  env:
+  - name: ANSIBLE_FORCE_COLOR
+    value: "True"
+  nodes:
+    edpm-compute-0:
+      hostName: edpm-compute-0
+      networks:
+      - name: ctlplane
+        subnetName: subnet1
+        defaultRoute: true
+        fixedIP: 192.168.122.100
+      - name: internalapi
+        subnetName: subnet1
+        fixedIP: 172.17.0.100
+      - name: storage
+        subnetName: subnet1
+        fixedIP: 172.18.0.100
+      - name: tenant
+        subnetName: subnet1
+        fixedIP: 172.19.0.100
+    edpm-compute-1:
+      hostName: edpm-compute-1
+      networks:
+      - name: ctlplane
+        subnetName: subnet1
+        defaultRoute: true
+        fixedIP: 192.168.122.101
+      - name: internalapi
+        subnetName: subnet1
+        fixedIP: 172.17.0.101
+      - name: storage
+        subnetName: subnet1
+        fixedIP: 172.18.0.101
+      - name: tenant
+        subnetName: subnet1
+        fixedIP: 172.19.0.101
+    edpm-compute-2:
+      hostName: edpm-compute-2
+      networks:
+      - name: ctlplane
+        subnetName: subnet1
+        defaultRoute: true
+        fixedIP: 192.168.122.102
+      - name: internalapi
+        subnetName: subnet1
+        fixedIP: 172.17.0.102
+      - name: storage
+        subnetName: subnet1
+        fixedIP: 172.18.0.102
+      - name: tenant
+        subnetName: subnet1
+        fixedIP: 172.19.0.102
+  nodeTemplate:
+    ansible:
+      ansiblePort: 22
+      ansibleUser: cloud-admin
+      ansibleVars:
+        timesync_ntp_servers:
+        - hostname: clock.redhat.com
+        edpm_network_config_hide_sensitive_logs: false
+        edpm_network_config_template: templates/single_nic_vlans/single_nic_vlans.j2
+        edpm_nodes_validation_validate_controllers_icmp: false
+        edpm_nodes_validation_validate_gateway_icmp: false
+        edpm_selinux_mode: enforcing
+        edpm_sshd_allowed_ranges:
+        - 192.168.122.0/24
+        edpm_sshd_configure_firewall: true
+        enable_debug: false
+    ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+  preProvisioned: true
+  tlsEnabled: true
+  secretMaxSize: 1920
```

Comment:
annotating for clarity here.  By setting the secretMaxSize to a small number (just over the size of cert +key), we simulate the case where more than one secret is needed to store the certs.

Reply:
The nodeset has 3 nodes, so we expect 3 cert/keys to be generated for generic-service1.  They won't all fit in one secret (to be mounted in the ansible EE container) because the secretMaxSize is so small.